### DISCUSSION TITLE
Access previous versions of a site

### DISCUSSION THREAD

**Discussion by synox** [adobe/helix-home#338] [Ideas]
Use cases: 
 - Let's say 3 pull requests were merged recently, and a new bug appears. Now I would like to check `when did it still work?`. 
 - Similarly, the lighthouse score dropped after 3 PRs, and I want to compare with an older version. 

Persona: 
Mostly developers. 

Suggestions: 
We could for example use a git hash to address a code revision. 
like https://c8ab36--vg-volvotrucks-us--hlxsites.hlx.page/

**Comment by tripodsan** [comment]
the git hash will not work, as we only sync branches. but you can use previous branches, as we don't delete them in the code-bus
